website/static/js/home-page/newAndNoteworthyPlugin.js

```diff
@@ -43,12 +43,17 @@ var NewAndNoteworthy = {
         var newAndNoteworthyUrl = $osf.apiV2Url('nodes/' + window.contextVars.newAndNoteworthy + '/node_links/', {});
         var newAndNoteworthyPromise = m.request({method: 'GET', url: newAndNoteworthyUrl, config: xhrconfig, background: true});
         newAndNoteworthyPromise.then(function(result){
-            var numNew = Math.min(result.data.length, self.SHOW_TOTAL);
+            var numNew = result.data.length;
             for (var l = 0; l < numNew; l++) {
                 var data = result.data[l];
                 if (lodashGet(data, 'embeds.target_node.data', null)) {
-                    self.newAndNoteworthyNodes().push(result.data[l]);
-                    self.fetchContributors(result.data[l]);
+                    if (data.embeds.target_node.data.attributes.public === true) {
```

Comment:
This is the case if we add a project to Discover Public Projects, and then that user makes the project private.  It is still in the list (because it is a node link), but now it's a private link.